setup.py

```diff
@@ -448,32 +470,77 @@ def main():
             os.path.join("/", "opt", "homebrew", "lib"),
             os.path.join("/", "opt", "homebrew", "opt"),
             os.path.join("/", "opt", "homebrew", "opt", "armadillo"),
-            os.path.join("/", "opt", "homebrew", "opt", "armadillo",
-                         "include"),
+            os.path.join(
+                "/", "opt", "homebrew", "opt", "armadillo", "include"
+            ),
             os.path.join(
                 "/", "opt", "homebrew", "opt", "armadillo", "include",
                 "armadillo_bits"
             ),
             # Needed for Mac Github Runners
             # for macos-10.15
-            os.path.join("/", "usr", "local", "Cellar", "libomp",
-                         "15.0.2", "include"),
+            os.path.join(
+                "/", "usr", "local", "Cellar", "libomp", "15.0.2", "include"
+            ),
             # for macos-latest
-            os.path.join("/", "usr", "local", "Cellar", "libomp",
-                         "15.0.7", "include"),
+            os.path.join(
+                "/", "usr", "local", "Cellar", "libomp", "15.0.7", "include"
+            ),
+        ]
+    else:  # WIN32
```

Comment:
Make this an explicit `elif` check for Windows, and add a new `else` that throws an `Exception("Unrecognized platform")`